docs/sop/WASABI_DATA_MIGRATION.md

```diff
@@ -0,0 +1,353 @@
+2023/01/18 11:37:10 DEBUG : Creating backend with remote "wasabi:gigadb-datasets/staging"
+2023/01/18 11:37:10 DEBUG : Using config file from "/root/.config/rclone/rclone.conf"
+    79710 Aldabra_Giant_Tortoise.png
+    39975 pub/10.5524/100001_101000/100002/CR.kegg.gz
+      649 pub/10.5524/100001_101000/100002/readme.txt
+     2696 pub/10.5524/100001_101000/100012/Keller.gff
+     3164 pub/10.5524/100001_101000/100012/readme.txt
+  5229469 pub/10.5524/100001_101000/100216/Badhwar_HBM_Brainspell-master.zip
+   618633 pub/10.5524/100001_101000/100217/Cipollini_HBM_NiData-master.zip
+   847985 pub/10.5524/100001_101000/100218/clark_mx15_bids-master.zip
+  2366391 pub/10.5524/100001_101000/100219/Craddock-AMX-Centrality-master.zip
+  1306938 pub/10.5524/100001_101000/100220/Das_HBM_LORIS-master.zip
+  1049584 pub/10.5524/100001_101000/100221/Dery_HBM_ClubsOfScience-master.zip
+```
+
+In addition, you can confirm the test dataset files have been uploaded into the
+Wasabi bucket by looking at the contents of `gigadb-datasets/staging` in the web
+console with your Wasabi subuser account.
+
+### Dataset file deletion
+
+To remove a dataset file from the `staging` directory:
+```
+# Do dry run first
+[gigadb@cngb-gigadb-bak]$ docker-compose run --rm rclone_cngb bash -c 'source /app/proxy_settings.sh; rclone --dry-run delete wasabi:gigadb-datasets/staging/pub/10.5524/100001_101000/100002/CR.kegg.gz'
+2023/01/18 12:45:38 NOTICE: CR.kegg.gz: Skipped delete as --dry-run is set (size 39.038Ki)
+
+# Delete file - looks like migration user cannot delete files from staging directory
+[gigadb@cngb-gigadb-bak]$ docker-compose run --rm rclone_cngb bash -c 'source /app/proxy_settings.sh; rclone delete wasabi:gigadb-datasets/staging/pub/10.5524/100001_101000/100002/CR.kegg.gz'
+```
+
+### Using migration tool to copy live data to Wasabi
+
+The `rclone_copy.sh` script can be used to copy `live` GigaDB data by adding the
+`--use-live-data` option as follows:
+```
+[gigadb@cngb-gigadb-bak]$ docker-compose run --rm rclone_cngb /app/rclone_copy.sh --starting-doi 100216 --ending-doi 100221 --use-live-data
+```
+
+This command will copy a set of real GigaDB datasets into the
+`gigadb-datasets/live` directory. You can confirm this by browsing to this 
+directory using the Wasabi web console as well as checking the latest log file 
+in the `logs` directory.
+
+If the `--use-live-data` flag is used and the script determines that it is not
+running on the CNGB server then the script will exit with an error message:
+```
+[centos@ip-xxx-xx-x-x]$ sudo docker-compose run --rm rclone /app/rclone_copy.sh --use-live-data --starting-doi 100002 --ending-doi 100020
+Cannot copy live data because we are not on backup server - exiting...
+```
+
+#### Batch size for file uploads
+
+The maximum number of datasets that can be uploaded has a default value of 100.
+This can be overridden using the `--max-batch-size`. For example, to increase
+the batch size to 200:
```

Comment:
Change `200` to `300` to match with the cmd below?

Reply:
@kencho51 This has been corrected now.